docs/toc.html

```diff
@@ -86,26 +86,26 @@
                         <li><a href="#basic_ops_increase_replication_factor">Increasing replication factor</a>
                     </ul>
                 <li><a href="#datacenters">6.2 Datacenters</a>
-		<li><a href="#georeplication">6.3 Geo-Replication (Cross-Cluster Data Mirroring)</a></li>
+                <li><a href="#georeplication">6.3 Geo-Replication (Cross-Cluster Data Mirroring)</a></li>
                     <ul>
-			<li><a href="#georeplication-overview">Geo-Replication Overview</a></li>
-			<li><a href="#georeplication-flows">What Are Replication Flows</a></li>
-			<li><a href="#georeplication-mirrormaker">Configuring Geo-Replication</a></li>
-			<li><a href="#georeplication-starting">Starting Geo-Replication</a></li>
-			<li><a href="#georeplication-stopping">Stopping Geo-Replication</a></li>
-			<li><a href="#georeplication-apply-config-changes">Applying Configuration Changes</a></li>
-			<li><a href="#georeplication-monitoring">Monitoring Geo-Replication</a></li>
+                        <li><a href="#georeplication-overview">Geo-Replication Overview</a></li>
+                        <li><a href="#georeplication-flows">What Are Replication Flows</a></li>
+                        <li><a href="#georeplication-mirrormaker">Configuring Geo-Replication</a></li>
+                        <li><a href="#georeplication-starting">Starting Geo-Replication</a></li>
+                        <li><a href="#georeplication-stopping">Stopping Geo-Replication</a></li>
+                        <li><a href="#georeplication-apply-config-changes">Applying Configuration Changes</a></li>
+                        <li><a href="#georeplication-monitoring">Monitoring Geo-Replication</a></li>
                     </ul>
-		<li><a href="#multitenancy">6.4 Multi-Tenancy</a></li>
+		        <li><a href="#multitenancy">6.4 Multi-Tenancy</a></li>
```

Comment:
I think this 6.4 line should align with 6.3 `li` tag, right? (also the following indent)

Reply:
You are right, on my IDE it was visually correct, but it was because that line contained 2 tabs, now they are replaced with spaces.